src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunnerFactory.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This RegexRunnerFactory class is a base class for compiled regex code.
+
+namespace System.Text.RegularExpressions
+{
+    abstract public class RegexRunnerFactory
```

Comment:
According to https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md, this should be `public abstract`.

This comment applies in other places.